client/js/yt-music/scroller.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * figures out which section the user is currently scrolled to
+ * in youtube-genre.html
+ *
+ * @returns {scroll}
```

Comment:
`@returns` tags need to have a description that specifies what you are returning, and the stuff in the curly braces is supposed to be a type (like `() => void` or `number`)